icu4c/source/test/cintltst/cloctst.c

```diff
@@ -259,6 +259,7 @@ void addLocaleTest(TestNode** root)
     TESTCASE(TestToLanguageTag);
     TESTCASE(TestBug20132);
     TESTCASE(TestBug20149);
+    TESTCASE(TestCLocale);
```

Comment:
nit: I'd probably suggest mentioning the word "default" in the test case name maybe.